CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # `include/checks.h` for details.
 if(MSVC)
   add_compile_options("/wd4244")
-  
+
```

Comment:
Since you're otherwise not touching this file, I would just revert this whitespace change.

Reply:
Done.